source/user-manual/reference/ossec-conf/cluster.rst

```diff
@@ -160,6 +161,233 @@ Toggles whether the cluster is enabled or not. If this value is set to **yes**,
+
++--------------------+-----------------------------------------+
+| **Default value**  | yes                                     |
++--------------------+-----------------------------------------+
+| **Allowed values** | yes, no                                 |
++--------------------+-----------------------------------------+
+
+
+.. _haproxy_address:
+
+haproxy_address
+^^^^^^^^^^^^^^^
+
+Specifies the address of HAProxy to communicate with.
+
++--------------------+--------------------------------------------------+
+| **Default value**  | n/a                                              |
++--------------------+--------------------------------------------------+
+| **Allowed values** | Any valid address (IP or DNS) of a cluster node. |
++--------------------+--------------------------------------------------+
+
+.. _haproxy_user:
+
+haproxy_user
+^^^^^^^^^^^^
+
+Specifies the user of HAProxy to connect with.
+
++--------------------+--------------------------------------------------+
+| **Default value**  | n/a                                              |
++--------------------+--------------------------------------------------+
+| **Allowed values** | Any valid username.                              |
++--------------------+--------------------------------------------------+
+
+.. _haproxy_password:
+
+haproxy_password
+^^^^^^^^^^^^^^^^
+
+Specifies the password of HAProxy to connect with.
+
++--------------------+--------------------------------------------------+
+| **Default value**  | n/a                                              |
++--------------------+--------------------------------------------------+
+| **Allowed values** | Any password.                                    |
++--------------------+--------------------------------------------------+
+
+.. _haproxy_port:
+
+haproxy_port
+^^^^^^^^^^^^
+
+Specifies the port to use for the HAProxy communication.
+
++--------------------+-------------------------------------+
+| **Default value**  | 5555                                |
++--------------------+-------------------------------------+
+| **Allowed values** | Any port number from 1024 to 65535. |
++--------------------+-------------------------------------+
+
+.. _haproxy_protocol:
+
+haproxy_protocol
+^^^^^^^^^^^^^^^^
+
+Specifies the protocol to use for the HAProxy communication.
+
++--------------------+-------------------------------------+
+| **Default value**  | http                                |
++--------------------+-------------------------------------+
+| **Allowed values** | http, https                         |
++--------------------+-------------------------------------+
+
+.. _haproxy_backend:
+
+haproxy_backend
+^^^^^^^^^^^^^^^
+
+Specifies the name of the backend that will be created in HAProxy.
+
++--------------------+-------------------------------------+
+| **Default value**  | wazuh_cluster                       |
```

Comment:
I will request changing this in the epic branch to `wazuh_reporting` since I find it less confusing (there are two backends/frontends: wazuh_register (1515) and wazuh_reporting (1514))
```suggestion
| **Default value**  | wazuh_reporting                       |
```